app/src/main/java/de/danoeh/antennapod/fragment/CompletedDownloadsFragment.java

```diff
@@ -322,6 +366,28 @@ public void onEndSelectMode() {
         swipeActions.attachTo(recyclerView);
     }
 
+    /**
+     * Returns the sort order for the downloads.
+     */
+    public SortOrder getDownloadsSortedOrder() {
+        String sortOrderStr = prefs.getString(PREF_DOWNLOADS_SORTED_ORDER, "use-default");
+        Log.d(TAG, sortOrderStr);
+        return SortOrder.parseWithDefault(sortOrderStr, SortOrder.DATE_NEW_OLD);
+    }
+
+    /**
+     * Sets the sort order for the downloads.
+     */
+    public void setDownloadsSortedOrder(SortOrder sortOrder) {
```

Comment:
No need to add the word "downloads" here - this is already in the downloadFragment class :)

Reply:
@ByteHamster I think I should move this back into the UserPreferences. Since I need to change `getDownloadedItems` to include a sortOrder parameter, this parameter needs to be accessible throughout the app. For example, there is a short download section on the home page that should also display the downloads in the order specified. Is it okay with you if I move this back? I ask because in a previous review you specifically asked me to move it to the CompletedDownloadsFragment.

Reply:
> For example, there is a short download section on the home page that should also display the downloads in the order specified.

Right. Okay, then let's keep it in UserPreferences